src/components/DividerBlock.ts

```diff
@@ -0,0 +1,12 @@
+import { Block, BlockProps } from './Block';
+import { DividerBlock as DividerBlockSpec } from '@slack/types';
+import { Childless } from './Childless';
+
+export type DividerBlockProps = BlockProps<'divider'> & Childless;
```

Comment:
is there a way to make use of the type (`divider` in this case) when setting the `type` in thee actual block?

Reply:
yeah see above comment... i was hoping to be able to do that... may have figured it out... check out the change to `ButtonElement`

Reply:
https://github.com/asynchronous-dev/react-chat-renderer/pull/16/commits/78f16540ab6a431dc951af0121d13b5c3c30fd20#diff-c6988041e186fcce0e7a2e8952375eceR22 thought it failed UTs.

Reply:
yeah i don't thin kthata works because reasons... 

* you have to make it optional otherwise the react component complains you're missing a required prop

<img width="705" alt="ScreenCapture at Wed Nov 13 11:20:13 PST 2019" src="https://user-images.githubusercontent.com/97470/68796357-999d9300-0607-11ea-9906-939d527313ec.png">

* if you make it optional, it doesn't seem to get populated by default... so UTs fail

<img width="688" alt="ScreenCapture at Wed Nov 13 11:23:49 PST 2019" src="https://user-images.githubusercontent.com/97470/68796643-1597db00-0608-11ea-9964-046cdcb9a128.png">

so maybe we should remove those hints?

Reply:
yeah if that worked it would make sense. but until it does work, it is probably excessive since `type` is already restricted by `@slack/types`. but 🤷‍♂ extra annotation isn't the worst thing if it may be useful

Reply:
yeah good point, the return type in `@slack` will ensure the safety there. i'll clean up for now.